Microsoft/microsoft-intune/ingest/parser.yml

```
@@ -9,7 +9,9 @@ pipeline:
       properties:
         input_field: "{{json_event.message.time}}"
         output_field: datetime
+
   - name: set_common_fields
+    filter: "{{json_event.message.category not in ['MicrosoftGraphActivityLogs', 'NonInteractiveUserSignInLogs']}}"
```

I suggest testing the category against the ones expected, not against the ones non-expected:
```suggestion
    filter: "{{json_event.message.category in ['AuditLogs', 'DeviceComplianceOrg', 'Devices', 'OperationalLogs']}}"
```